ouroboros-network/src/Ouroboros/Network/Connections/Client/Inet.hs

```diff
@@ -0,0 +1,37 @@
+{-# LANGUAGE DataKinds #-}
```

Comment:
Inet is a rather poor name given that we're supporting UNIX domain sockets too right now, and probably windows named pipes in the future.

Reply:
Changed to `Socket/Client.hs` and `Socket/Server.hs`